pkg/syncer/util.go

```diff
@@ -247,23 +245,32 @@ func isValidvSphereVolumeClaim(ctx context.Context, pvcMetadata metav1.ObjectMet
 	return false
 }
 
-// isValidvSphereVolume returns true if the given PV metadata of a vSphere
-// Volume (in-tree volume) and has migrated-to annotation on the PV,
+// isValidvSphereVolume returns true if the given PV is of a vSphere
+// Volume (in-tree volume),
 // or if the PV was provisioned by CSI driver using in-tree storage class.
-func isValidvSphereVolume(ctx context.Context, pvMetadata metav1.ObjectMeta) bool {
+func isValidvSphereVolume(ctx context.Context, pv *v1.PersistentVolume) bool {
```

Comment:
As discussed, some of the changes to this doc needs to be reverted. Please take a look

Reply:
we discussed reverting the change made for isValidvSphereVolumeClaim.
fixed it.

Reply:
fixed.